js/main.js

```diff
@@ -158,7 +157,7 @@ function editClick(event) {
         submitForm.elements.title.value = data.entries[i].titleValue;
         submitForm.elements.url.value = data.entries[i].urlValue;
         submitForm.elements.notes.value = data.entries[i].notesValue;
-        submitForm.elements.entryId = data.editing;
+        submitForm.elements.entryId = data.entries[i].entryId;
```

Comment:
What is this line? There shouldn't be a form control element for the entry id, you make those yourself using the values in the data model 🤔